cmds/mknod/mknod_test.go

```diff
@@ -76,22 +70,14 @@ func TestInvocationErrors(t *testing.T) {
 	}
 
 	for _, v := range tests {
-		c := exec.Command(mknodPath, v.args...)
+		c := testutil.Command(t, v.args...)
 		_, e, _ := run(c)
 		if e[20:] != v.expects {
 			t.Errorf("mknod for '%v' failed: got '%s', want '%s'", v.args, e[20:], v.expects)
 		}
 	}
 }
 
-func TestMknodBlock(t *testing.T) {
-	uid := syscall.Getuid()
-
-	if uid != 0 {
-		t.Logf("not root, uid %d, skipping test\n", uid)
-		return
-	}
-	t.Log("root user, proceeding\n")
-	//TODO(ganshun): implement block test
```

Comment:
@GanShun What were you planning for this test? Something like `mknod test b 8 1`?